Tmain/omit-long-patterns-etags.d/stdout-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+input.sh,349
+                                                                                                func96(func961,0
```

Comment:
I don't see it. There are just as many spaces as in the source file (and if truncated, the function name would either be missing or at least incomplete, in the first field).